### DISCUSSION TITLE
Documentation Handinlg Webhook calls from payment gateway ?

### DISCUSSION THREAD

**Discussion by neerajgupta2407** [saleor/saleor#7580] [General]
HI,

I am imlementing new payment gateway.
I have implemented standard function for plugin ( authorize, process_payment, void, refund, capture.)

Now i want to implement webhook( or Event) calls which payment gateway sent on any payment updates.

Is there any documentation available for handling webhook/Event calls for payment gateway  ?

Thanks in Advance

**Comment by codecompagnons** [comment]
I activated Stripe plugin already. In test modus it is working

**Comment by korycins** [comment]
https://docs.saleor.io/docs/3.0/developer/extending/plugins#webhook-handler
